packages/react-components/src/components/SendBox.tsx

```diff
@@ -85,6 +90,11 @@ export interface SendBoxStrings {
    * Aria label for send message button
    */
   sendButtonAriaLabel: string;
+  /* @conditional-compile-remove(file-sharing) */
+  /**
+   * Error message indicating that all file uploads are not complete.
+   */
+  fileUploadsPendingError: string;
```

Comment:
Do we need this as a prop for sendbox as this is a standard message(Uploading..please wait)

Reply:
We allow contoso to override all the strings. They might want to display a different error message.

Reply:
Also, this is the only way we can localize the string.